internal/handlers/tigris/msg_drop.go

```diff
@@ -46,11 +46,18 @@ func (h *Handler) MsgDrop(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 		return nil, err
 	}
 
-	if err = h.driver.UseDatabase(db).DropCollection(ctx, collection); err != nil {
-		if status.Code(err) == codes.NotFound {
+	err = h.driver.UseDatabase(db).DropCollection(ctx, collection)
+	switch err := err.(type) {
+	case nil:
+		// do nothing
+	case *driver.Error:
+		//nolint:nosnakecase // Tigris named their const that way
+		if err.Code == api.Code_NOT_FOUND {
```

Comment:
Maybe extract it into `isNotFound` function to have `//nolint` in a single place?

Reply:
fixed